static/js/modal.js

```diff
@@ -0,0 +1,86 @@
+var Modal = (function () {
+    var funcs = {
+        createModal: function () {
+            var overlay = document.createElement("overlay");
+            overlay.className = "overlay new";
+            overlay.setAttribute("data-overlay", "custom-modal-" + Math.random().toString(16));
```

Comment:
We can just use a counter (in a module-level variable) instead of random, and not have to worry about collisions, right?